bin/key_server.js

```diff
@@ -174,6 +175,11 @@ function main() {
       })
     }
   })
+  .catch((err) => {
+    log.error({ op: 'server.start.1', msg: 'startup caught error',
```

Comment:
We can skip the log, since this is already logged in the `run` function, at https://github.com/mozilla/fxa-auth-server/blob/master/bin/key_server.js#L118.

Seems like all we need to do in `main` is just exit on rejection.